### PR TITLE
Added Quoted args parsing

### PR DESCRIPTION
**Pull request by stijnb1234**
You can now add `@ConcatQuotes` to the function to make it parse quotes on all the arguments.

I also updated some dependencies.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -59,17 +59,22 @@
         <dependency>
             <groupId>org.reflections</groupId>
             <artifactId>reflections</artifactId>
-            <version>0.9.10</version>
+            <version>0.9.12</version>
```
Comment (Kaktushose) [inline pom.xml:62]: this needs to stay on 0.9.10, since 0.9.12 has some bugs

File: pom.xml [hunk 2]
```diff
@@ -59,17 +59,22 @@
...
         <dependency>
             <groupId>com.google.code.gson</groupId>
             <artifactId>gson</artifactId>
-            <version>2.8.6</version>
+            <version>2.8.8</version>
+        </dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+            <version>30.1.1-jre</version>
```
Comment (Kaktushose) [inline pom.xml:77]: what is guava needed for?

**Comment by Kaktushose** [comment]
Thanks for you contribution. I'm currently working on version 2.0.0, see #32, so I'll just merge this without requesting any changes. The dev branch is a total mess and undeployable atm, however I'll gladly use your code for v2 :)

File: pom.xml (hunk 2 above)
Comment (stijnb1234) [reply pom.xml:77]: I got building issues without guava, for example the IO classes were missing. By adding guava, those imports could be found again.

Comment (Kaktushose) [reply pom.xml:77]: hmm, strange. Doesn't matter that much atm, will sort such things before I release 2.0.0

**Pull request merged**